### PR TITLE
Fix output dir, add quiet mode.

### PR DESCRIPTION
**Pull request by dschneller**
See commit comments for the reasoning behind these changes.

### CODE REVIEW

**Comment by maxhq** [comment]
Thank you very much for your improvements!
I did not have time to do an in-depth review yet, but as soon as that's done, I'll merge them into my repo!

**Comment by endersonmaia** [comment]
LGTM :+1:

File: zabbix-mysql-dump [hunk 1]
```diff
@@ -189,33 +220,50 @@ while read table; do
...
-	mysqldump --routines --opt --single-transaction --skip-lock-tables \
-		$dump_opt $MYSQL_CONN --tables ${table} >>"${DUMPFILE}"
-
-	# show percentage
-	i=$((i+1)); i_percent=$(($i * 100 / $DB_TABLE_NUM))
-	if [ $(($i_percent % 12)) -eq 0 ]; then
-		echo -n "${i_percent}%"
-	else
-		if [ $(($i_percent % 2)) -eq 0 ]; then echo -n "."; fi
+	TABLE_DUMP_CMD="mysqldump ${MYSQL_CONN} --routines --opt --single-transaction --skip-lock-tables $dump_opt $DBNAME --tables ${table}"
```
Comment (endersonmaia) [inline zabbix-mysql-dump]: I'm getting this error when not using the conf file.

```
# ./zabbix-mysql-dump -n -h 127.0.0.1 -u zabbix -pzabbix -d zabbix -o /root/
Configuration:
 - host:     127.0.0.1 (127.0.0.1)
 - database: zabbix
 - user:     zabbix
Fetching list of existing tables...
Starting table backups...
mysqldump: Couldn't find table: "zabbix"
Failed dumping table acknowledges. Aborting.
````

I found that $DBNAME is already on ${MYSQL_CONN}

The TABLE_DUMP_CMD is like this `mysqldump -h 127.0.0.1 -u zabbix -pzabbix zabbix --routines --opt --single-transaction --skip-lock-tables --no-data zabbix --tables acknowledges`

Where `zabbix` appears two times, and the command tries to find a table named `zabbix`, and ends with an error.

**Comment by maxhq** [comment]
@endersonmaia Thank you for your comments! Sorry I left this request open for so long. Splitting the patch up would indeed help me, as the fault you pointed out has to be fixed.
The config file handling introduced in 9d3cfcd6e56c2de01e5a3907ac1213e33ff15eac is a nice idea, but I think it should be adjusted. Currently it just leaves out a fixed set of options from being read from the command line - I think command line options should always win (just like mysql handles it).

**Comment by maxhq** [comment]
I incorporated all changes but had to rewrite quite some of the code.
Thank you @dschneller for your contributions and @endersonmaia + @sm4sh1k for your comments!

The new release is available here: https://github.com/maxhq/zabbix-backup/releases/tag/v0.8.0

**Pull request closed without merging**